src/components/infomodel-structure/model-element-list/index.tsx

```diff
@@ -22,9 +22,9 @@ const ModelElementList: FC<Props> = ({ title, properties, modelElements }) =>
   properties && properties.length > 0 ? (
     <>
       <ListTitleSC.ListTitle>{title}</ListTitleSC.ListTitle>
-      {properties.map(property => (
+      {properties.map((property, index) => (
         <Element
-          key={property.identifier}
+          key={property.identifier ?? `property-${index}`}
```

Comment:
du kan bruke `property.uri ?? property.identifier`
Minst en av de to skal ha verdi, men sikkert greit å ha med index, sånn i tilfelle

Reply:
Ikke med nåværende typer som er satt opp for denne komponenten. Den tar imot liste enten med ModelCodeElement-er eller ModelProperty-er. Eneste krysningspunktet er identifier. Så endringen vil kreve litt mer arbeid.